### PR TITLE
Blazor content

### PR DESCRIPTION
**Pull request by csharpfritz**

### CODE REVIEW

**Comment by csharpfritz** [comment]
Tagging @jongalloway @katiesavage and @jamesmontemagno for review

File: 5-blazor/README.md [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+
+- Remove the default layout and extra pages in the application to make it smaller
+- Improve the parameters to the `Board` component so that you can pass any valid CSS color value.
+-	Improve the indicators appearance with some CSS and HTML layout
+- Introduce sound effects
+- Add a visual indicator and prevent a drop button from being used when the column is full
+- Add networking capabilities so that you can play a friend in their browser
+- Insert the game into a .NET MAUI with Blazor application and play it on your phone or tablet.
+
+Happy coding and have fun!
```
Comment (katiesavage) [inline 5-blazor/README.md:530]:
```suggestion

Happy coding and have fun!

## Connect with us

We're excited to support you on your learning journey! Check out the [.NET Community Page](https://dotnet.microsoft.com/platform/community) to find links to our blogs, YouTube, Twitter, and more.
```

**Comment by Humoyunmirzo-coder** [comment]
Hello, can I participate in your project?

**Comment by katiesavage** [comment]
@Humoyunmirzo-coder of course! Learn how to go through the content by looking at the instructions in the [README](https://github.com/dotnet/intro-to-dotnet-web-dev/blob/main/README.md)

**Comment by claudedjale** [comment]
Hello,

I took a break because I figured that I needed to learn/understand .Net Framework first before starting to learn its features. I will resume my training soon, but I can say that I understand ASPNET core better than when I started.

Thanks,
Claude

Claude Djale

Support Engineer

Microsoft Azure Technical Support
Office:  +1 (425) 7046444 | ***@***.***
Working hours: 9:00AM to 6:00PM PST
Customer Service and Support | Azure App Services - Core Chat

[cid:1db5da20-fb97-4bfc-911d-3865e387f383]
Microsoft Azure<http://azure.microsoft.com/> | Azure Status<http://azure.microsoft.com/en-us/status/> | Support Plans<http://azure.microsoft.com/en-us/support/plans/> | Create a Case<http://azure.microsoft.com/en-us/support/options/> | Privacy Policy<https://privacy.microsoft.com/en-us/PrivacyStatement>

If you would prefer to work with an Azure support engineer in another time zone or if you need assistance outside of my working hours, please forward this email to ***@***.*** with a brief explanation.  One of my colleagues will gladly continue working on this issue.

Managerial Assistance: ***@***.***
***@***.***?anonymous&ep=signature>            Book time to meet with ***@***.***?anonymous&ep=signature>
________________________________

**Pull request merged**